lib/fast-forward.js

```diff
@@ -117,12 +112,25 @@ module.exports = class FastForward {
     if (this.destroyed) return false
 
     this.entropy = this.system.entropy
+    this.encryptionLength = this.system.encryptionLength
+
+    if (this.core.manifest.version >= 2) {
+      // technically diverges from system.views to include encryption here, but makes sense
+      this.views.push({
+        key: this.core.manifest.linked[0],
+        length: this.system.encryptionLength
+      })
+    }
 
     const promises = []
 
     // ensure local key is locally available always
     promises.push(this.system.get(this.base.local.key, { timeout: this.timeout }))
 
+    if (this.encryption && this.encryption.core) {
```

Comment:
also gueard that the encrypionLength is > 0 ?

Reply:
since we call _waitForAppend on this core above it would already have timed out in that case

Reply:
it does imply that we always have to populate the encryption view when we make it default (pretty sure we don't that atm, but ff tests will catch that when the time comes and there's no easy way to force the manifest version so we can test atm)